tests/test_assessment.py

```diff
@@ -1908,6 +1920,142 @@ def test_can_get_multiple_assessment_items(self):
         self.assertEqual(data[0]['id'], item1_id)
         self.assertEqual(data[1]['id'], item2_id)
 
+    def test_can_get_assessment_item_qti(self):
+        mc_sentence = self.create_mw_sentence_item()
+
+        assessments_endpoint = self.url + '/assessments'
+
+        # Use POST to create an assessment
+        assessment_name = 'a really hard assessment'
+        assessment_desc = 'meant to differentiate students'
+        payload = {
+            "name": assessment_name,
+            "description": assessment_desc,
+            "itemIds": [mc_sentence['id']]
+        }
+        req = self.app.post(assessments_endpoint,
+                            params=json.dumps(payload),
+                            headers={'content-type': 'application/json'})
+        self.ok(req)
+        assessment_id = unquote(json.loads(req.body)['id'])
+        assessment_detail_endpoint = assessments_endpoint + '/' + assessment_id
+        assessment_items_endpoint = assessment_detail_endpoint + '/items?qti'
+
+        # should now appear in the Assessment Items List
+        req = self.app.get(assessment_items_endpoint)
+        self.ok(req)
+        data = self.json(req)
+        self.assertEqual(len(data), 1)
+        self.assertIn('qti', data[0])
+
+        item = data[0]
+
+        item_qti = BeautifulSoup(item['qti'], 'lxml-xml').assessmentItem
+        expected_values = ['id51b2feca-d407-46d5-b548-d6645a021008',
+                           'id881a8e9c-844b-4394-be62-d28a5fda5296',
+                           'idcccac9f8-3b85-4a2f-a95c-1922dec5d04a',
+                           'id28a924d9-32ac-4ac5-a4b2-1b1cfe2caba0',
+                           'id78ce22bf-559f-44a4-95ee-156f222ad510',
+                           'id3045d860-24b4-4b30-9ca1-72408a3bcc9b',
+                           'id2cad48be-2782-4625-9669-dfcb2062bf3c']
+        for index, value in enumerate(item_qti.responseDeclaration.correctResponse.find_all('value')):
+            self.assertEqual(value.string.strip(), expected_values[index])
+
+        item_body = item_qti.itemBody
+
+        order_interaction = item_body.orderInteraction.extract()
+
+        audio_asset_label = 'ee_u1l01a01r05__mp3'
+        image_asset_label = 'medium8081173379968782030intersection_png'
+        audio_asset_id = item['question']['fileIds'][audio_asset_label]['assetId']
+        audio_asset_content_id = item['question']['fileIds'][audio_asset_label]['assetContentId']
+        image_asset_id = item['question']['fileIds'][image_asset_label]['assetId']
+        image_asset_content_id = item['question']['fileIds'][image_asset_label]['assetContentId']
+
+        expected_string = """<itemBody>
```

Comment:
A question: Testing against expected HTML markup seems fragile.  Is the markup created within the test?  I don't find it but perhaps it's not obvious.